packages/x-date-pickers/src/DateCalendar/DateCalendar.types.ts

```diff
@@ -78,7 +78,7 @@ export interface ExportedDateCalendarProps<TDate extends PickerValidDate>
   /**
    * Component displaying when passed `loading` true.
    * @returns {React.ReactNode} The node to render when loading.
-   * @default () => <span data-mui-test="loading-progress">...</span>
+   * @default () => <span data-testid="loading-progress">...</span>
```

Comment:
This is technically a lie. 🙈 
`data-testid` is never present in a bundle users end up with. 🙈 
```suggestion
   * @default () => <span>...</span>
```

Reply:
`data-mui-test` didn't too, idk why it was there. I just did a search and replace.

I will remove them all from docs pages

Reply:
Yes, precisely. 👍 
That's why I'm saying that such JSDoc and API description is simply misleading. 🙈